spec/rails_helper.rb

```diff
@@ -230,10 +230,91 @@
       "Arlington, TX"=>[32.7355816, -97.1071186],
       "Cleveland, OH"=>[41.4996574, -81.6936772]
     }
+
     @lat_min_range = -0.29
     @lat_max_range = 0.29
     @long_min_range = -0.37
     @long_max_range = 0.37
```

Comment:
@DavisWeimer Do we need this data in the `rails_helper` file anymore?

Reply:
Good point, this data really is only needed for CSV gen. I can remove it and if we need to reference it, it is on the CSV branch.